src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$circle-diametr: 500px;
+$dot-diametr: 15px;
+$start-pos: 0 + 180deg;
+$end-pos: 180 + 360deg;
```

Comment:
Why do you use those strange angles? Try to find the reason.
The right way is to start from 0deg and finish at 360deg (1turn).